internal/cmd/request.go

```diff
@@ -115,3 +123,71 @@ func PrintCMDResponseFull(request any, response core.APIResponse, responseType a
 		fmt.Println(util.MustToJSONIndent(responseContent))
 	}
 }
+
+// Attempt to convert an API response to a TSV table.
+// Returns false if there are issues or the TSV table if it was successful.
+func PrintCMDResponseTable(response core.APIResponse) (string, bool) {
+	responseContent, ok := response.Content.(map[string]any)
+	if !ok {
+		return "", false
+	}
+
+	// Don't try to format a response that has multiple keys.
+	if len(responseContent) != 1 {
+		return "", false
+	}
+
+	responseContentKey := reflect.ValueOf(responseContent).MapKeys()
+
+	// Get the rows that will be added to the table.
+	entries, ok := responseContent[responseContentKey[0].String()].([]any)
+	if !ok {
+		return "", false
+	}
+
+	// Use the first entry to create the headers of the table.
+	firstEntry, ok := entries[0].(map[string]any)
+	if !ok {
+		return "", false
+	}
+
+	var headers []string
+	for key := range firstEntry {
+		headers = append(headers, key)
+	}
+
+	sort.Strings(headers)
+
+	var customTable strings.Builder
+	customTable.WriteString(strings.Join(headers, "\t"))
+
+	headerKeys := strings.Split(customTable.String(), "\t")
+
+	customTable.WriteString("\n")
+
+	// Turn each entry into a row of the table.
+	for i, entry := range entries {
+		entryMap, ok := entry.(map[string]any)
+		if !ok {
+			return "", false
+		}
+
+		var row []string
+		for _, key := range headerKeys {
+			switch value := entryMap[key].(type) {
+			case map[string]any:
+				row = append(row, util.MustToJSON(value))
+			default:
+				row = append(row, fmt.Sprintf("%v", value))
+			}
+		}
+
+		customTable.WriteString(strings.Join(row, "\t"))
+
+		if i < len(entries)-1 {
```

Comment:
I don't like the no space around binary operators (here, `-`) and having a non-leaf expression in a binary comparison (here, `<`) without parens.

By doing the latter, gofmt will let you do the former.